posthog/api/decide.py

```diff
@@ -83,7 +83,9 @@ def get_decide(request: HttpRequest):
     if request.method == "POST":
         try:
             data = load_data_from_request(request)
-        except RequestParsingError as error:
+            api_version_string = request.GET.get("v")
+            api_version = int(api_version_string) if api_version_string else 1  # TODO: semantic_version
```

Comment:
It's a future concern imo — if and when we get more granular, like `2.1.0` or whatever, casting to an int and using regular math operators will no longer be enough.

Reply:
got it - that makes sense!

suggestion: maybe switch it to a `# NOTE: this does not support semantic versioning e.g. 2.1.0` 

since right now we don't actually have plans or a need todo anything here? I just don't want it to look like something important was forgotten when someone else is reviewing the code down the road